job-dsls/jobs/osbs/osbs_build_pipeline.jenkinsfile

```diff
@@ -0,0 +1,230 @@
+@Library(['jenkins-pipeline-shared-libraries','qe-library']) _
+
+Map<String, String> buildResults = [:]
+
+Map<String, String> imageBuildReferences = [:]
+Map<String, String> operatorBuildReferences = [:]
+
+String propertyFileUrl = "${CI_MESSAGE}"
+
+TIMEOUT = 4
+
+pipeline {
+    options {
+        timeout(time: TIMEOUT, unit: 'HOURS')
+    }
+
+    agent {
+        label 'kie-rhel7 && !built-in'
```

Comment:
maybe user rhl8?

Reply:
using the same labels as in the most pipelines in this repository, maybe it would be better to do the update for the whole repo.
For this script it shouldn't be a problem to run on rhel8.